gcsa/google_calendar.py

```diff
@@ -212,6 +212,27 @@ def add_quick_event(
         ).execute()
         return EventSerializer.to_object(event_json)
 
+    def import_event(self, event, **kwargs):
+        """Imports an event in the calendar
```

Comment:
Could you please add this line to the docs here?

    This operation is used to add a private copy of an existing event to a calendar.

So it's more clear what it does.

Reply:
Sure. But do you mean the docstring right here, where you commented? The .rst docs? Both?

Reply:
I meant docstring here. But now I'm thinking it would be useful in `.rst` docs too. So both. Thanks!

Reply:
okok, chief.